dynatrace-aws-logs.sh

```diff
@@ -204,7 +204,7 @@ EOF
   if [ -z "$TARGET_API_TOKEN" ]; then echo "No --target-api-token"; print_help_deploy; exit 1; fi
```

Comment:
Again, here I mean 169 - wouldn't that fail the deployment with no param given?

Reply:
Exactly, those checks are done in case the argument is present in the first place. There's a case for that, line 154

Reply:
I see, thanks! I thought it would be something like that as it would be hard to miss when testing :P